### PR TITLE
Add dependencies for meta-analysis tutorial

### PR DESCRIPTION
**Pull request by tsalo**
This PR adds NiMARE and matplotlib as dependencies for the meta-analysis tutorial.

Related to https://github.com/neurohackademy/nh2021-curriculum/pull/4.

### CODE REVIEW

File: deployments/hub-neurohackademy-org/image/Dockerfile [hunk 1]
```diff
@@ -129,10 +129,12 @@ RUN echo "Installing pip packages..." \
         indexed-gzip \
         ipyvolume==0.6.0a8 \
         line_profiler \
+        matplotlib \
```
Comment (consideRatio) [inline deployments/hub-neurohackademy-org/image/Dockerfile]:
```suggestion
```

matplotlib is already installed from the base image as listed here: https://github.com/pangeo-data/pangeo-docker-images/blob/master/pangeo-notebook/packages.txt

Comment (tsalo) [reply deployments/hub-neurohackademy-org/image/Dockerfile]: 🤦 sorry about that! I'll remove it then.

Comment (consideRatio) [reply deployments/hub-neurohackademy-org/image/Dockerfile]: No worries at all, a technical and contextual detail!

**Comment by consideRatio** [comment]
@tsalo we have run into an upstream issue that makes this build fail. I'll try resolve it and then merge this PR.

Reported it upstream in repo2docker: https://github.com/jupyterhub/repo2docker/issues/1065, PR to fix it in https://github.com/jupyterhub/repo2docker/pull/1066.

**Comment by tsalo** [comment]
Thank you!

**Pull request merged**